lib/json/static.nit

```diff
@@ -152,56 +146,36 @@ redef class FlatText
 	end
 end
 
-redef class Int
-	super Jsonable
-end
-
-redef class Float
-	super Jsonable
-end
-
-redef class Bool
-	super Jsonable
-end
-
 # A map that can be translated into a JSON object.
-interface JsonMapRead[K: String, V: nullable Jsonable]
+interface JsonMapRead[K: String, V: nullable Serializable]
 	super MapRead[K, V]
-	super Jsonable
+	super Serializable
 end
 
 # A JSON Object.
 class JsonObject
```

Comment:
virtual type on Object is an hack and have issues as is. I prefer to wait to solve there issues.